api/app/dao/callback_failures_dao.py

```diff
@@ -28,6 +29,33 @@ def dao_get_callback_failures_by_service_id(service_id):
     return CallbackFailure.query.join(CallbackFailure.notification).filter_by(service_id=service_id)
 
 
+def dao_get_callback_failures_for_all_services(page=1):
+    return CallbackFailure.query.join(
+        Notification, CallbackFailure.notification_id == Notification.id
```

Comment:
is this join needed / used anywhere? technically this column refers to `NotificationHistory` and not `Notification` however it does work if needed.

Reply:
Yep not needed